### ISSUE TITLE
Feature Request: Share History notification

### ISSUE THREAD

**Issue by GarethPrice-Aus** [galaxyproject/galaxy#7802]
Would it be possible to have some form of notification when a new History is shared with me?
Specifically thinking under User or beside the "Histories Shared with Me" an exclamation mark, an icon of some description to notify me a new history has been shared with me.
Alternatively email notification of a new shared History would be great but I understand that is contingent on the particular user authorising email contact and the particular Galaxy service being able to email.
It is simply that acting as core service and promoting Galaxy as an analysis destination I don't always know when collaborators / clients / students have shared work with me and can only monitor this by periodic checking of "Histories Shared with Me"

Thanks, Gareth

**Comment by hexylena** [comment]
xref https://github.com/galaxyproject/galaxy/issues/2324, definitely an issue that's bothered me for a while :)

**Comment by mvdbeek** [comment]
Agreed. Another way to tackle this is to have a github style activity area where we'd list the most recent histories, workflows, workflow invocations and jobs.

**Comment by hexylena** [comment]
@mvdbeek think that could definitely work as an implementation of 2324, good suggestion/comparison, I like it better than my proposed facebook style one.

**Comment by GarethPrice-Aus** [comment]
Great and thanks @mvdbeek and @erasche for both taking this seriously. And double thanks for not slamming the door with a "this has already been raised"!

**Comment by davelopez** [comment]
This should be already covered by #15663 :)

![Notifications](https://user-images.githubusercontent.com/46503462/246444471-3b6d0cd4-cde1-4663-ac3f-d0de1502a2d3.png)
